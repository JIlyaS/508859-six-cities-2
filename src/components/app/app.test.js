import React from 'react';
import renderer from 'react-test-renderer';
import App from './app';

jest.mock(`../map/map`);

it(`App correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<App
      offers={[{
        id: `id0`,
        title: ``,
        type: `private room`,
        price: 0,
        img: `room.jpg`,
        rating: 0,
        isPremium: false,
        coordinate: [],
        photos: [],
        features: [],
        insideProperties: [],
        hostUser: {
          avatar: ``,
          name: ``,
          status: ``,
          description: []
        },
        reviews: [{
          id: `id0`,
          avatar: ``,
          name: ``,
          rating: 0,
          date: ``,
          description: ``
        }]
      }]}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
