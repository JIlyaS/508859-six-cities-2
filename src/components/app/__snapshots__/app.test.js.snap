// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders after relaunch 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/offer/:offerId"
  />
</Switch>
`;
